libarchive/archive_read_support_format_iso9660.c

```diff
@@ -1218,6 +1218,7 @@ archive_read_format_iso9660_read_header(struct archive_read *a,
 				    "No memory");
 				return (ARCHIVE_FATAL);
 			}
+			memset(iso9660->utf16be_previous_path, 0, UTF16_NAME_MAX);
```

Comment:
The better fix is to change the `malloc` call above to use `calloc` instead.  (`calloc` gives you memory that's already been zeroed, and some allocators can do this faster than `malloc` + `memset`).)  It would probably be a good idea to look at other uses of `malloc` in this file and see if they should also use `calloc`.